basin3d/plugins/usgs.py

```diff
@@ -258,7 +263,7 @@ def _load_point_obj(datasource, json_obj, feature_observed_properties, synthesis
             )],
             # geographical_group_id=huc_accounting_unit_id,
             # geographical_group_type=FeatureTypeEnum.REGION,
-            observed_property_variables=observed_property_variables,
+            observed_property_variables=observed_property_variables[id],
```

Comment:
I think we should do some error handling for this element of the monitoring_feature object in the case that somehow the site ("id") is not in the observed_property_variables dict. Something like adding a if statement before the object is build to check if "id" is in the keys of the dict. If not, set the value to "Could not find observed property variables for this monitoring feature." This element is a nice to have but the object shouldn't fail to build if if is not there.